packages/ibm-products/src/components/CoachmarkOverlayElements/CoachmarkOverlayElements.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { Children, useEffect, useRef, useState } from 'react';
+import { Button } from 'carbon-components-react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import { useCoachmark } from '../Coachmark';
+import { clamp } from 'lodash';
+
+//TODO THIS PATH WILL NEED TO BE UPDATED ONCE IN IBM PRODUCTS
+import { Carousel } from '../Carousel';
+//TODO THIS PATH WILL NEED TO BE UPDATED ONCE IN IBM PRODUCTS
+import { SteppedAnimatedMedia } from '../SteppedAnimatedMedia';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg /*, carbon */ } from '../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--coachmark-overlay-elements`;
+const componentName = 'CoachmarkOverlayElements';
+
+const defaults = {
+  nextButtonLabel: 'Next',
+  previousButtonLabel: 'Back',
+  closeButtonLabel: 'Got it',
+};
+
+/**
+ * Composable container to allow for the displaying of CoachmarkOverlayElement
+ * components in a carousel fashion.
+ */
+export let CoachmarkOverlayElements = React.forwardRef(
+  (
+    {
+      className,
+      children,
+      media,
+      nextButtonLabel = defaults.nextButtonLabel,
```

Comment:
In other components, we have tended to use naming such as `nextButtonText` rather than `nextButtonLabel`. Are you ok with that change?

Reply:
I think we pulled that from the ExampleComponent (primaryButtonLabel, secondaryButtonLabel) - totally can change that though